From 025dbca3e49e2f94479d3865f10e1689ff817687 Mon Sep 17 00:00:00 2001
From: Philipp Zabel <p.zabel@pengutronix.de>
Date: Fri, 12 Feb 2016 14:21:27 +0100
Subject: [PATCH] kmssink: enable UYVY, YUY2, and YVYU (interleaved YUV 4:2:2)
 formats

https://bugzilla.gnome.org/show_bug.cgi?id=761059
(cherry picked from commit 2f51985191aeeb384c853eba563dd54726aa7fac)
---
 sys/kms/gstkmsutils.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/sys/kms/gstkmsutils.c b/sys/kms/gstkmsutils.c
index e7f605d..ddf8d2d 100644
--- a/sys/kms/gstkmsutils.c
+++ b/sys/kms/gstkmsutils.c
@@ -53,8 +53,9 @@ static const struct
   DEF_FMT (ABGR8888, ABGR),
   DEF_FMT (XBGR8888, xBGR),
 #endif
-  /* DEF_FMT (YUYV, ???), */
-  /* DEF_FMT (YVYU, ???), */
+  DEF_FMT (UYVY, UYVY),
+  DEF_FMT (YUYV, YUY2),
+  DEF_FMT (YVYU, YVYU),
   DEF_FMT (YUV420, I420),
   DEF_FMT (YVU420, YV12),
   DEF_FMT (YUV422, Y42B),
-- 
1.9.1

